lib/ts/index.ts

```diff
@@ -91,6 +92,52 @@ export default class AuthHttpRequest {
             userContext: getNormalisedUserContext(input === undefined ? undefined : input.userContext)
         });
     };
+
+    static getInvalidClaimsFromResponse = async function(input: {
+        response: { data: any } | Response;
+        userContext?: any;
+    }): Promise<ClaimValidationError[]> {
+        return AuthHttpRequestFetch.recipeImpl.getInvalidClaimsFromResponse({
+            response: input.response,
+            userContext: getNormalisedUserContext(input.userContext)
+        });
+    };
+
+    static validateClaims = (
+        overrideGlobalClaimValidators?: (
+            globalClaimValidators: SessionClaimValidator[],
```

Comment:
```suggestion
            globalClaimValidators: SessionClaimValidator[] | Promise<SessionClaimValidator[]>,
```

Reply:
We decided to keep this synchronous cause it makes it easier for us to implement SSR